src/groups/rotation_action.jl

```diff
@@ -0,0 +1,39 @@
+@doc doc"""
+    RotationAction(M::Manifold, SOn::SpecialOrthogonal)
+
+Space of actions of the [`SpectialOrthogonal`](@ref) group `\mathrm{SO}(N)`
+on a Euclidean-like manifold `M` of dimension `N`.
+"""
+struct RotationAction{TM<:Manifold,TSO<:SpecialOrthogonal} <: AbstractActionOnManifold
```

Comment:
I believe in the earlier issue we had discussed that there was no canonical action of a rotation group on a manifold, with the example we discussed being action of `SO(2)` on `ℝ³`. The methods below would apply just fine if typed on something like `RotationAction{Euclidean{Tuple{N}},SpecialOrthogonal{N}}`, or in fact, any `Euclidean` where the first dimension is `N`.

Reply:
I was thinking about using this action in three cases:
1. action of SO(n) on R^n,
2. action of SO(n) on [landmark manifolds on R^n](https://academic.oup.com/blms/article-abstract/16/2/81/325787?redirectedFrom=fulltext),
3. action of SO(n) on manifolds of curves on R^n.

I should probably tighten method signatures here.

Reply:
I don't know anything about landmark manifolds on R^n, but the other two sound good. I think additionally you could have action of SO(n) on SO(n).

Reply:
These don't all necessarily need to be in this PR.

Reply:
I've opened an issue for this: #76